setup.py

```diff
@@ -1,17 +1,37 @@
 from setuptools import setup
 
+with open('LONG_DESCRIPTION.rst') as f:
+    long_description = f.read()
+
 setup(
     name='ably',
-    version='0.1.dev',
+    version='0.0.2',
```

Comment:
Why `0.0.2`?  Is this our version number?  If so, it should be `0.8.0` and we need to add a tag `v0.8.0` to the Git repo here on Github as well, @fjsj can you do that?